gitlab-pipeline/stage/init.yml

```diff
@@ -140,13 +140,13 @@ init:workspace:
         git fetch -u -f origin $META_MENDER_REV:pr &&
         git checkout pr ||
         git checkout -f -b pr $META_MENDER_REV
-      )
+      ) || return 1
     - (
-        cd meta-mender
-        echo "# $(git log -n1 --oneline)" >> ${CI_PROJECT_DIR}/build_revisions.env
-        echo "export META_MENDER_REV=$META_MENDER_REV" >> ${CI_PROJECT_DIR}/build_revisions.env
+        cd meta-mender &&
+        echo "# $(git log -n1 --oneline)" >> ${CI_PROJECT_DIR}/build_revisions.env;
+        echo "export META_MENDER_REV=$META_MENDER_REV" >> ${CI_PROJECT_DIR}/build_revisions.env;
         echo "export META_MENDER_REV_GIT_SHA=$(git rev-parse HEAD)" >> ${CI_PROJECT_DIR}/build_revisions.env
-      )
+      ) || return 1
```

Comment:
Not wrong, but I believe a better fix here is to prefix everything with `-`, so that it remains a list. This is not only less error prone because it doesn't require semicolons, but also looks better in the log.

Reply:
Makes sense, thank you. I did copied it from the above function but here it makes sense to be multi-line.

Reply:
Like this? The first block I had to leave as one line because it makes use of `&&` and `||` to control what to execute.

Reply:
Exactly! Thanks!